docs/source/modin/supported/index_supported.rst

```diff
@@ -77,11 +77,11 @@ Methods
 +-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
 | ``delete``                  | N                               |                                  |                                                    |
 +-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
-| ``drop``                    | N                               |                                  |                                                    |
+| ``drop``                    | D                               |                                  |                                                    |
```

Comment:
for the following default to pandas cases, I believe we don't have any tests for those. Let's raise NotImplementedError instead.

Reply:
actually, @sfc-gh-azhan i do not expect we have any default to pandas cases, we probably should remove that D from all supported api.rst, right?

Reply:
right. The code actually call `to_pandas` to implement those APIs we should raise NotImplementedError instead.